src/Microsoft.Azure.SignalRService/ServiceHubContext.cs

```diff
@@ -24,5 +26,11 @@ public Task DisposeAsync()
         {
             throw new NotImplementedException();
         }
+
+        internal ServiceHubContext(IHubContext<Hub> hubContext, IHubLifetimeManagerForUserGroup lifetimeManager)
```

Comment:
No need to make the constructor internal since the class is internal. make it public